database/schema.sql

```diff
@@ -0,0 +1,54 @@
+create table if not exists item (
+  id  INT generated always as identity,
+  label_id int,
+  genre_id int,
+  author_id int,
+  publish_date date,
+  archived boolean,
+  primary key(id),
+  foreign key (label_id) references labels (id),
+  foreign key (genre_id) references genre (id),
+  foreign key (author_id) references authors (id)
+);
+
+CREATE TABLE MUSIC_ALBUM(
+  on_spotify boolean NOT NULL,
+  id int GENERATED BY DEFAULT AS IDENTITY,
+  publish_date date NOT NULL,
+  archived boolean
+) INHERIT(item);
+
+CREATE TABLE genre(
+  id int GENERATED BY DEFAULT AS IDENTITY,
+  name varchar(255)
+);
+
+
+create table Game (
+  multiplayer BOOLEAN,
+  last_played_at DATE
+)
+
+create table Authors (
+  id  INT generated always as identity,
+  first_name varchar(500),
+  last_name varchar(500),
+  primary key(id),
+
+  foreign key(id) references item(id)
```

Comment:
- [ ] Good job working on the schema queries but please kindly ensure you include indexes for all foreign keys.